bundles/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -114,3 +132,29 @@ then
 end 
 
 ```
+
+dwdunwetter_severity.map
+
+```
+Minor=Wetterwarnung
+Moderate=Markante Wetterwarnung
+Severe=Unwetterwarnung
+Extreme=Extreme Unwetterwarnung
+NULL=undefiniert
```

Comment:
We should amend a mapping for `UNDEF` too:

```suggestion
NULL=undefiniert
UNDEF=undefiniert
```

Reply:
Thanks... Changed and fixed